### PR TITLE
Adds infiltrators to dynamic

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

It is a strange and not cool gamemode, where:
1. All infiltrators die at the start of the game, and then the crew has a long boring shift without any antagonists
2. Infiltrators kill everyone
3. Infiltrators stealthy complete their objectives and then the crew has a long boring shift without anything interesting

So i remove the infiltration gamemode from rotation, but instead enable infiltration event and add infiltration as a dynamic ruleset(roundstart and midround). Now infiltrators are side antagonists instead of being main gamemode heroes.

# Changelog

:cl: SuperSlayer
rscadd: Added infiltrators as a roundstart dynamic ruleset
rscadd: Added infiltrators as a midround dynamic ruleset
tweak: Enabled infiltrator event
rscdel: Infiltrators gamemode is removed from rotation
/:cl:

### CODE REVIEW

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
Fix your title
Don't remove the gamemode from rotation if you believe it should be in dynamic
Don't add it as an event

**Comment by ghost** [comment]
> Fix your title Don't remove the gamemode from rotation if you believe it should be in dynamic Don't add it as an event

Why? Inflitrators aren't good as a gamemode heroes, but it could be okay if infiltrators would be secondary antagonists, and because of this i turned them here from a gamemode to a dynamic ruleset/event.

**Comment by Cartlord** [comment]
Gamerdk, why do you think it shouldn't be an event?

**Comment by redmoogle** [comment]
my only gripe with this is if someone steals the infils objectives and now they have to try to find it out of x amount of people

**Comment by ghost** [comment]
> my only gripe with this is if someone steals the infils objectives and now they have to try to find it out of x amount of people

This can happend to... almost any other antagonist, so I don't think that it is an issue

**Comment by redmoogle** [comment]
> > my only gripe with this is if someone steals the infils objectives and now they have to try to find it out of x amount of people
> 
> This can happend to... almost any other antagonist, so I don't think that it is an issue

well *generally* theres only very little overlap in objectives unless your running dynamic

**Comment by JamieD1** [comment]
Add it to dynamic but dont disable it as a gamemode.

**Comment by ghost** [comment]
Okay

**Comment by ghost** [comment]
Enabled infiltrators in config

File: yogstation/code/modules/events/infiltrators.dm [hunk 1]
```diff
@@ -2,13 +2,13 @@
 	name = "Infiltrators"
 	typepath = /datum/round_event/ghost_role/infiltrators
 
-	weight = 0
+	weight = 8
```
Comment (Bibby0110) [inline yogstation/code/modules/events/infiltrators.dm]:
```suggestion
	weight = 0
```
since we're keeping it as a gamemode, no event

**Review by Bibby0110 — APPROVED** [review]

File: config/game_options.txt [hunk 2]
```diff
@@ -106,7 +106,7 @@ ALERT_DELTA Destruction of the station is imminent. All crew are instructed to o
 # New
 PROBABILITY DARKSPAWN 6
 PROBABILITY HERESY 4
-PROBABILITY INFILTRATION 4
+PROBABILITY INFILTRATION 2
```
Comment (JamieD1) [inline config/game_options.txt]:
```suggestion
PROBABILITY INFILTRATION 4
```

**Pull request merged**